src/smartContractQueriesController.ts

```diff
@@ -21,14 +20,19 @@ interface ILegacyQuery {
     getEncodedArguments(): string[];
 }
 
+interface IQueryRunner {
+    networkProvider: INetworkProvider;
```

Comment:
This should not be needed. Generally speaking, now, the queries controller should not know about any network provider.

Reply:
Indeed, removed